frontend_tests/node_tests/stream_list.js

```diff
@@ -638,7 +638,7 @@ run_test('update_count_in_dom', () => {
     stream_li.addClass('stream-with-count');
     assert(stream_li.hasClass('stream-with-count'));
 
-    const stream_count = new Dict();
+    const stream_count = new IntDict();
```

Comment:
I was sorta surprised that the prior commits still worked without this commit, but I guess the code under test didn't really care whether the dict was Dict or IntDict, since the interfaces are the same.  If it's not too hard to squash these changes in, though, it would probably make for more cohesive commits.

Reply:
`stream_count` in the actual code has been an `IntDict` since your commit 9ba1829243fbd726b9347d864ecd9dc807bbd0aa, so it’s too late for squashing.